.github/project.yml

```diff
@@ -2,4 +2,6 @@ name: Java Operator SDK Extension
 release:
   current-version: 7.0.0.CR1
   next-version: 7.0.0-SNAPSHOT
-
+  quarkus-platform:
+    target-branch: main
+    needs-main-pr: false
```

Comment:
I guess this shouldn't be needed anymore if we specify branches to target explicitly